app/controllers/v1/people_controller.rb

```diff
@@ -1,7 +1,7 @@
 class V1::PeopleController < V1::BaseController
 
   def create
-    @person = PersonWithRemoteFields.find_or_build(person_params)
+    @person = PersonConstructor.new(person_params).build
```

Comment:
It seems like PersonConstructor.new is never called without build.  whatever build returns should probably be a part of the response of new.